ruby-challenges.rb

```diff
@@ -33,4 +104,37 @@
 # Expected output example: my_bike.brake(5) => 23
 # Expected output example: my_bike.brake(25) => 0
 
-# Pseudo code:
\ No newline at end of file
+# Pseudo code:
+#       input: 
+#       output:
+#       process: I know i need to add method to our class Bike
+#                pedal_faster & brake
+#                pedal faster needs to increase speed by given amount
+#                brake should decrease by given amount
+#                integer for argument
+#                cannot be neg int on brake portion so must stop at 0
+#                we need to set conditions for brake speed so its my favorite problem solving bff, conditionals!
+
+
+# class Bike
+#     def initialize(model, wheels, current_speed)
+#         @model = model 
+#         @wheels = wheels
+#         @current_speed = current_speed
+#     end
+# # 
+#     def pedal_faster 
```

Comment:
When refactoring think about what is recquired to see a increase in speed if the speed starts at 0